src/Authentication/SignUp.js

```diff
@@ -0,0 +1,22 @@
+import { FcGoogle } from 'react-icons/fc';
```

Comment:
Why add this in this folder? Make it jsx and make signup folder in component and add